### ISSUE TITLE
Link to latest Snapshot is broken

### ISSUE THREAD

**Issue by marszuck** [th-otto/AtariX#1]
The link for downloading the latest snapshots seems broken.

**Comment by th-otto** [comment]
Yes, that's because they are not available since bintray has closed. I have to convert the scripts first.
